packages/schematics/angular/application/files/src/index.html.template

```diff
@@ -7,6 +7,311 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link
```

Comment:
I feel users might just forget these here when removing the demo code.

I tried moving them into `src/styles.css` like this:
```
@import "https://fonts.googleapis.com/css?family=Roboto";
@import "https://fonts.googleapis.com/css?family=Roboto+Mono";
@import "https://fonts.googleapis.com/icon?family=Material+Icons";
```

And it does work, in Chrome at least. But it's not converted to anything else. It relies on native browser support for `@import` in CSS. This it wouldn't be a great choice either. Might just be better to keep it here.

Reply:
I see your point about it not being obvious for users to delete. Like with the rest of the CSS I like having it all in one place to just focus on one file when removing template code.

What if we add comments for now that say "remove this template code"? I'll implement that for now until we come up with something better.

Reply:
Added the comments `<!-- *** Remove Template Code Below *** -->` and `<!-- *** Template Code Ends *** -->` in both template and in index files.

Reply:
That SGTM, let's roll with it.